app/views/shared/_title.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="callout <%= callout.callout_class %>">
+  <div class="callout-title">
+    <%= callout.callout_title %>
```

Comment:
For clarity, let's either call the variable `callout_presenter` or drop the `callout_` from it's attr_readers.